validator-node-setup.md

```diff
@@ -0,0 +1,157 @@
+# 1. Running a Node
+Here we describe how to run and sync a node using celestia-appd.
```

Comment:
We need a Table of Content 
If you are using vscode, you can use this plugin to generate the ToC 
https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one

Reply:
Thanks, I added an index for the whole doc :)